orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -503,48 +503,40 @@ void resize_impl_ort_(
 
 namespace aten {
 
-at::Tensor empty_memory_format(
+at::Tensor empty_strided(
   at::IntArrayRef size,
-  // *,
+  at::IntArrayRef stride,
   c10::optional<at::ScalarType> dtype_opt,
-  c10::optional<at::Layout> layout_opt,
-  c10::optional<at::Device> device_opt,
-  c10::optional<bool> pin_memory,
-  c10::optional<at::MemoryFormat> memory_format) {
-  ORT_LOG_FN(size, dtype_opt, layout_opt, device_opt, pin_memory, memory_format);
-
-  assert(dtype_opt.has_value());
-  assert(device_opt.has_value());
+  c10::optional<at::Layout> layout_opt,  // Ignored because there's no ONNX support.
+  c10::optional<at::Device> device_opt,  // Will be ORT by the time this is dispatched.
+  c10::optional<bool> pin_memory_opt) {  // Ignored because there's no ONNX support.
+  ORT_LOG_FN(size, stride, dtype_opt, layout_opt, device_opt, pin_memory_opt);
 
-  // TODO: validate options and memory format
-  // TODO: figure out how to get the correct element type.
   OrtValue ot;
+  assert(device_opt.has_value());
+  at::ScalarType dtype = c10::dtype_or_default(dtype_opt);
```

Comment:
we are now using a default type if it is null, the assert behavior was suspicious to me, especially since once impl asserted and the other used default.  I'll see if I can test it

Reply:
tested this, found that I couldn't get dtype_opt to not have a value